plugins/home/backstage-plugin-home-rss/src/RSSCard/Content.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import {Progress, ErrorPanel, Table} from '@backstage/core-components';
+import { RSSContentProps} from './types';
+import {useAsync} from "react-use";
+import {Box} from "@material-ui/core";
+import axios from 'axios';
+
+type DataItem = {
+  title: any
+}
+
+const columns = [{
+  title: "", field: "title"
+}]
+
+/**
+ * A component to render a RSS feed
+ *
+ * @public
+ */
+export const Content = (props: RSSContentProps) => {
+  const [error, setError] = useState<Error | undefined>();
+  const [data, setData] = useState<DataItem[]>([]);
+  const parser = new DOMParser();
+  const [title, setTitle] = useState<string | undefined>();
+
+  useAsync(async () => {
+    if (error) { return }
+    if (data.length > 0) { return }
+    const headers = new Headers({
+      "Accept": "application/rss+xml"
+    });
+    try {
+      const response = await axios.get(props.feedURL, {headers: headers});
```

Comment:
Upstream decided to not use axios, I think we should follow it in our plugins. So in the frontend plugins we could just use the native `fetch`.